openpype/pipeline/__init__.py

```diff
@@ -194,5 +208,5 @@
 
     # Backwards compatible function names
     "install",
-    "uninstall",
+    "uninstall"
```

Comment:
We add them so a future PRs which add after or move it does not have to add commit to the line.